src/data/faq.json

```diff
@@ -129,8 +129,10 @@
                                 "anchor": "risk_update_24",
                                 "active": false,
                                 "textblock": [
+                                    "<b>Updated 30 March, 2022</b>",
                                     "The Corona-Warn-App updates the risk status at least every 24 hours. The actual time can lag behind for a few hours from the time of the day before.",
-                                    "Since CWA version 1.7, exposure checks are run up to 6 times per day when a Wi-Fi connection is present. The repeated daily check is performed automatically. While using mobile data, the check is only performed once every 24 hours due to the <a href='#mobile_data_costs' target='_blank' >zero rating commitment</a> of the network providers."
+                                    "Since CWA version 1.7, exposure checks are run up to 6 times per day when a Wi-Fi connection is present. The repeated daily check is performed automatically. While using mobile data, the check is only performed once every 24 hours due to the <a href='#mobile_data_costs' target='_blank' >zero rating commitment</a> of the network providers.",
+                                    "Starting with version 2.20, the Corona-Warn-App notifies you when the date of the last encounter with increased risk on the red tile has changed. Previously, you were not notified again if they had another, more recent encounter with increased risk."

```

Comment:
"..you were not notified again if they had another, ..." =>
"you were not notified again if you had another, "